integration/spark/src/test/scala/org/apache/carbondata/view/rewrite/TestPartitionWithMV.scala

```diff
@@ -748,6 +748,22 @@ class TestPartitionWithMV extends QueryTest with BeforeAndAfterAll with BeforeAn
     sql("drop table if exists partitionone")
   }
 
+  test("test partition on MV with sort column") {
+    sql("drop table if exists partitionone")
+    sql("create table if not exists partitionone (ts timestamp, " +
+        "metric STRING, tags_id STRING, value DOUBLE) partitioned by (ts1 timestamp,ts2 timestamp) stored as carbondata TBLPROPERTIES ('SORT_COLUMNS'='metric,ts2')")
```

Comment:
I think above MV rearrange code was added when partition column is not a sort column case I guess.

can you please add one more test case, same but just remove sort columns from the create table and test it ?

Reply:
All other testcase in TestPartitionWithMV are without sort column only and also tested manually. It works